pilot/lib/pilot/build_manager.rb

```diff
@@ -96,6 +96,15 @@ def distribute(options, build: nil)
       build.auto_notify_enabled = config[:notify_external_testers]
 
       return if config[:skip_submission]
+      if options[:reject_build_waiting_for_review]
+        waiting_for_review_build = Spaceship::TestFlight::Build.all_waiting_for_review(app_id: build.app_id, platform: :ios).first
```

Comment:
Should this be `platform: fetch_app_platform`?

Reply:
@joshdholtz Good catch. I have made the required change. 👍